pkg/query-service/app/logparsingpipeline/controller.go

```diff
@@ -273,6 +308,13 @@ func (pc *LogParsingPipelineController) RecommendAgentConfig(
 		return nil, "", apiErr
 	}
 
+	// recommend default pipelines along with user created pipelines
+	defaultPipelines, err := pc.getDefaultPipelines()
```

Comment:
so we are adding this pipeline by default, i.e we are adding this to all our old customers by default right ? 

Ideally we wanted to add this to all user over time.